### PR TITLE
New hardware_arduino layer 

### PR DESCRIPTION
**Pull request by Suryansh470**
It is still in progress...

### CODE REVIEW

File: hardware_layer/hardware_arduino/src/thruster.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        return LOW;
+    else if(pwm>=180 && pwm<220)
+        return MED;
+    else
+        return HIGH;
+}
+
+void thruster::setValues(int HC, int MC, int LC, int HAC, int MAC, int LAC)
+{
+    clock_.High=HC;
```
Comment (Mayankm96) [inline hardware_layer/hardware_arduino/src/thruster.cpp]: To make expressions look clean, it is recommended that spaces are used often. For instance, `clock_.High = HC;`

Comment (Suryansh470) [reply hardware_layer/hardware_arduino/src/thruster.cpp]: Done

**Comment by Suryansh470** [comment]
@Mayankm96  only one Cmake file is left from our side rest everything is done !!! 👍  I will complete and push it by tomorrow. !! 👍

**Comment by Mayankm96** [comment]
Is this code working? I am closing this pull request since it has become too old. Please send in a new one once it's completed and is working.

**Pull request closed without merging**